app/file-upload/server/lib/ranges.js

```diff
@@ -0,0 +1,49 @@
+export function getByteRange(header) {
+	if (!header) {
+		return;
+	}
+	const matches = header.match(/(\d+)-(\d+)/);
+	if (!matches) {
+		return;
+	}
+	return {
+		start: parseInt(matches[1], 10),
```

Comment:
This may return `infinity` for veeery big numbers. Is that ok?

Reply:
This was being used before here: https://github.com/RocketChat/Rocket.Chat/pull/21463/files#diff-6b22c83cdb33e48bba19cad98bb830a3e373cf4abd6a488f523884a5b2c63cf5L59

so I'm not that worried about it

Reply:
Nice! 🥳